core/primitives/src/views.rs

```diff
@@ -1672,16 +1672,21 @@ pub struct TxStatusView {
     serde::Deserialize,
     Clone,
     Debug,
+    Default,
     Eq,
     PartialEq,
+    Ord,
+    PartialOrd,
 )]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
 pub enum TxExecutionStatus {
     /// Transaction is waiting to be included into the block
     None,
     /// Transaction is included into the block. The block may be not finalised yet
-    Inclusion,
+    Included,
```

Comment:
This change might be dangerous, but we merged this enum only few weeks ago, and it's not used in production yet.
I realised that it's better if all the options would be adjectives, it's more intuitive to see "included" near "executed"